tools/capable.py

```diff
@@ -86,25 +88,37 @@
 #include <linux/sched.h>
 
 struct data_t {
-   // switch to u32s when supported
-   u64 pid;
-   u64 uid;
+   u32 tgid;
+   u32 pid;
+   u32 uid;
    int cap;
    int audit;
    char comm[TASK_COMM_LEN];
+#ifdef KERNEL_STACKS
+   int kernel_stack_id;
+#endif
 };
 
 BPF_PERF_OUTPUT(events);
 
+#ifdef KERNEL_STACKS
+BPF_STACK_TRACE(stacks, 1024);
+#endif
+
 int kprobe__cap_capable(struct pt_regs *ctx, const struct cred *cred,
     struct user_namespace *targ_ns, int cap, int audit)
 {
-    u32 pid = bpf_get_current_pid_tgid();
+    u64 __pid_tgid = bpf_get_current_pid_tgid();
+    u32 __tgid = __pid_tgid >> 32;
+    u32 __pid = __pid_tgid;
```

Comment:
I suspect the original intention is for tgid, so __pid here may not really be needed. Could you double check?

Reply:
True. What about printing both, PID and TID, like we do with other tools? I can add that.

Reply:
Yes, we can print both.